indicators/NO_ALIE_001-004/R/NO_ALIE_001-004.qmd

```diff
@@ -0,0 +1,821 @@
+
+## 6. References
+
+Artsdatabanken (2023, 11. august). Fremmede arter i Norge - med økologisk risiko 2023. http://www.artsdatabanken.no/lister/fremmedartslista/2023.
+
+Framstad, E., Kolstad, A. L., Nybø, S., Töpper, J. & Vandvik, V. 2022. The condition of forest and mountain ecosystems in Norway. Assessment by the IBECA method. NINA Report 2100. Norwegian Institute for Nature Research.
+
+Tingstad, L., Evju, M., Sickel, H., & Töpper, J. 2019. Utvikling av nasjonal arealrepresentativ naturovervåking (ANO). Forslag til gjennomføring, protokoller og kostnadsvurderinger med utgangspunkt i erfaringer fra uttesting i Trøndelag. NINA Rapport 1642.
+
+
+## 7. Datasets
+
+### 7.1 ANO monitoring data
+
+ANO stands for areal-representativ naturovervåking, i.e. area representative nature monitoring. 1000 sites are randomly distributed across mainland Norway and visited within a 5-year cycle. Each ANO site spans a 500 x 500 m grid cell, and the data collection at each ANO site includes up to 12 evenly distributed 250 sqm monitoring circles (up to 12, because some of these evenly distributed points may be in water or otherwise inaccessible). The cover of each alien plant species in the circle is recorded, in addition to the ecosystem the circle lies in, including ecosystem type and some additional variables related to ecosystem-specific drivers of state. In the analyses described in this document, we use the data available on Miljødirektoratets kartkatalog (https://kartkatalog.miljodirektoratet.no/Dataset/Details/2054), which comprises data from the first round of data collection in ANO, 2019-2024, and a total of 18 871 circles in 1052 sites.
+The timeframe of six years from 2019-2024 allows to create indicator values for the two 3-year reporting periods 2019-2021 and 2022-2024.
+
+## 8. Spatial units
+
+The spatial units for the 'area without alien plant species' indicator are governed by the ANO data, which consist of 1000 randomly chosen sites in Norway. Each site is a 500 x 500 m grid cell with 18 monitoring circles of 250sqm and a central 1 x 1 m vegetation plot. The basic unit for which observational alien plant species abundance data exist, and for which every the 'area without alien plant species' indicator is computed, is the 250sqm circle. Aggregation of these basic spatial units to higher ones - like sites, municipalities, counties, regions, or the national level - should be done with appropriate consideration of the spatial structure of the data, i.e. imbalances and clustering (highly unequal number of points of an ecosystem type between sites).
+
+## 9. Analyses
+
+Import testing data:
+```{r}
+#Sys.setlocale("LC_ALL", "no_NB.utf8") #works with æøå or use "Norwegian"
+# ANO data
+ANO.sp <- st_read("C:/Users/joachim.topper/OneDrive - NINA/work/projects/oekologisk tilstand/Naturovervaking_eksport.gdb",
```

Comment:
same problem here, I cannot access the data and therefore not render the qmd

Reply:
@joatop, kan du kanskje legge dataene på `P:\41201785_okologisk_tilstand_2022_2023`?